### ISSUE TITLE
"Search and Navigate directly to a skill" does not announce when no results are found

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#2261]

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/2271

**Comment by dwalizer** [comment]
:+1:
